drivers/mem_manage/mm_drv_common.c

```diff
@@ -290,30 +290,26 @@ int sys_mm_drv_move_region(void *virt_old, size_t size, void *virt_new,
 		uintptr_t pa = phys_new + offset;
 		uint32_t flags;
 		int ret2;
-		bool to_map;
 
-		to_map = true;
 		ret2 = z_sys_mm_drv_page_flag_get(va_old, &flags);
 		if (ret2 != 0) {
 			ret = ret2;
-			to_map = false;
-		}
-
-		ret2 = sys_mm_drv_unmap_page(va_old);
-		if (ret2 != 0) {
-			ret = ret2;
-		}
-
-		if (!to_map) {
+		} else {
 			/*
-			 * Cannot retrieve flags of mapped virtual memory.
-			 * Skip mapping this page as we don't want to map
-			 * with unknown random flags.
+			 * Only map the new page when we can retrieve
+			 * flags of the old mapped page as We don't
+			 * want to map with unknown random flags.
```

Comment:
Are the changes to `drivers/mem_manage/mm_drv_common.c` in commit `drivers: mem_manage: add driver for Intel Audio DSP` a rebasing error?

Reply:
I was at the wrong commit when refreshing using stgit. Fixed.